packages/turf-line-split/test.js

```diff
@@ -33,6 +33,16 @@ test('turf-line-split', t => {
     t.end();
 });
 
+test('turf-line-split -- lines should split the same feature 1 with 2 as 2 with 1', t => {
+    const geojson = load.sync(directories.in + 'linestrings.geojson');
+    const featureOne = geojson.features[0];
+    const featureTwo = geojson.features[1];
+
+    const resultsOne = lineSplit(featureOne, featureTwo);
+    const resultsTwo = lineSplit(featureTwo, featureOne)
+    t.deepEquals(resultsOne, resultsTwo, 'Splits were made in different locations')
```

Comment:
`resultsOne` will never equal to `resultsTwo`

Reply:
Ah, you're right... 

I need to be comparing the intersection points. Sorry, that was foolish of me! In my head I was comparing the points that the lines were split at, but of course this isn't doing that!!

Reply:
:) No worries, your test case's intent is valid, the intersections should be compared.